config/serialization/Pay.xml

```diff
@@ -29,6 +29,12 @@
         <attribute name="encodedCardData">
             <group>commerce_weavers_sylius_tpay:shop:order:pay</group>
         </attribute>
+        <attribute name="saveCard">
+            <group>commerce_weavers_sylius_tpay:shop:order:pay</group>
+        </attribute>
+        <attribute name="savedCardId">
+            <group>commerce_weavers_sylius_tpay:shop:order:pay</group>
+        </attribute>
         <attribute name="tpayChannelId">
```

Comment:
I wonder, do we need `saveCard` and `saveCardId`. If `saveCardId` is null then we know `saveCard` is false. Or I am missing something?

Reply:
it is `saveCard` and `save**D**CardId`. One is a boolean flag to save encodedData other contains ID of credit card from customer. Boolean flag have sense only when send together with `encodedData`